corehq/apps/users/models.py

```diff
@@ -3080,4 +3081,6 @@ def role(self):
                 return UserRole.get(self.role_id)
             except ResourceNotFound:
                 logging.exception('no role with id %s found in domain %s' % (self.role_id, self.domain))
+        elif self.domain:
+            return CouchUser.from_django_user(self.user).get_domain_membership(self.domain).role
```

Comment:
This looks like a behavior change. Curious why it was included in this PR?

Reply:
Yeah, I agree, I should drop that. I'm probably going to amend this commit instead of adding a new one, so that when we merge the original branch we get a conflict instead of the risk of dropping these two lines by accident. (I'm not sure I explained that very well. And I think we'd get a conflict either way. But I'd hate Git to apply the "drop those lines" commit to the second merge.)

(I also forgot to rename that migration.)